metrics/cli/test/services/sentry/adapter/sentry_cli_service_adapter_test.dart

```diff
@@ -150,35 +147,35 @@ void main() {
     );
 
     test(
-      ".createRelease() prompts the user to enter the project slug",
+      ".getSentryRelease() prompts the user to enter the project slug",
       () async {
         whenPromptReleaseName().thenReturn(releaseName);
 
-        await sentryService.createRelease([sourceMap]);
+        sentryService.getSentryRelease();
 
         verify(prompter.prompt(enterProjectSlug)).called(once);
       },
     );
 
     test(
-      ".createRelease() throws if prompter throws during the project slug prompting",
+      ".getSentryRelease() throws if prompter throws during the project slug prompting",
       () {
         whenPromptProjectSlug().thenThrow(stateError);
 
         expect(
-          sentryService.createRelease([sourceMap]),
+          () => sentryService.getSentryRelease(),
           throwsStateError,
         );
       },
     );
 
     test(
-      ".createRelease() stops the release creation process if prompter throws during the project slug prompting",
+      ".getSentryRelease() stops the config prompting process if prompter throws during the project slug prompting",
       () async {
         whenPromptProjectSlug().thenThrow(stateError);
 
-        await expectLater(
-          sentryService.createRelease([sourceMap]),
+        expect(
+          () => sentryService.getSentryRelease(),
```

Comment:
🔨
```suggestion
          sentryService.getSentryRelease,
```
What do you think?